Vagrantfile

```diff
@@ -225,10 +237,11 @@ def get_domains(config)
   subdomains = %w( www webgrind adminer mailcatcher browsersync info )
 
   subdomains.each do |domain|
-    domains << "#{domain}.#{config['name']}.dev"
+    domains << "#{domain}.#{config['name']}.local"
   end
 
-  domains << config['name']+".seravo.dev" #test https-domain-alias locally
+  domains << config['name']+".seravo.local" # test https-domain-alias locally
```

Comment:
why do we need this ? shouldn't it be `*.wp-palvelu.local` anyway? usually you just configure the domains you need to use for this site in the config.yml, for which the example file already contains this entry. See: https://github.com/Seravo/wordpress/blob/master/config-sample.yml#L13